Yoast/wordpress-seo
# Increased coverage of indexable post builder, indexable term builder and social image trait

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Improves the coverage of the indexable post- and term builder and the social image trait.

## Relevant technical choices:

* Decided to remove the `find_alternative_image` and `set_alternative_image` methods from the `Indexable_Post_Builder`, since they are the same as the ones it inherits from the social image trait.
* Refactors setting the permalink on the post indexable in the post builder to its own method, to make testing it a bit easier.

## Test instructions

This PR can be tested by following these steps:

* Run the unit tests and check that the code coverage of `src/builders/indexable-term-builder.php`, `src/builders/indexable-post-builder.php` and `src/builders/indexable-social-image-trait.php` are 100%.
* Check that the Travis build passes.

## UI changes

## Documentation

## Quality assurance


Fixes #
